training/src/components/SelectField/index.js

```diff
@@ -1,3 +1,3 @@
 import SelectField from './SelectField';
```

Comment:
instead of this you can simple use

export { default as SelectField } from './SelectField';